tests/testthat/test-download-data-cube.r

```diff
@@ -0,0 +1,36 @@
+local_path <- "../testdata"
+
+test_that("File is downloaded",{
+  #Download May 2020 EQ09
+  local_path <- "../testdata"
+
+  download_abs_data_cube(cat_no = "6291.0.55.003",
+                         cube = "EQ09", date = "May 2020",
+                         path = local_path)
+
+  expect_equal(file.exists(file.path(local_path, "eq09.zip")), TRUE)
+})
+
+
+test_that("File can be unzipped",{
+  local_path <- "../testdata"
```

Comment:
Should be a `tempfile()` for CRAN.

Reply:
I have changed to `tempdir()` - which I think should work?